modules/markdown.php

```diff
@@ -6,7 +6,7 @@
  * Sort Order: 31
  * First Introduced: 2.8
  * Requires Connection: No
- * Auto Activate: No
+ * Auto Activate: Yes
```

Comment:
Am I wrong in that this is the only thing you'll need to do to make it activate by default for new sites?